File: opentsdb-proxy/src/main/java/net/opentsdb/proxy/clients/EchoClient.java
package net.opentsdb.proxy.clients;

import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

public class EchoClient implements Client {
  private final static Logger logger = LoggerFactory.getLogger(EchoClient.class);

  @Override
  public void sendMessage(String message) {
    logger.info("Got message: {}", message);
  }
}

File: opentsdb-proxy/src/main/scala/net/opentsdb/proxy/clients/EchoClient.scala
package net.opentsdb.proxy.clients

import org.slf4j.LoggerFactory

class EchoClient extends Client{
  private final val logger = LoggerFactory.getLogger(classOf[EchoClient])

  def sendMessage(message: String) {
    logger.info("Got message: {}", message)
  }
}
